### ISSUE TITLE
Brave: Log in with passkey -> Verify with biometrics errors if browser extension is already unlocked

### ISSUE THREAD

**Issue by ethansaxenian** [bitwarden/clients#9795]
### Steps To Reproduce

1. Ensure the browser extension is already unlocked
2. Attempt to log into a website (e.g. https://community.bitwarden.com) with a passkey
3. Click the `Confirm` button on the `Log in with passkey?` popup

### Expected Result

I expect to be logged into the website via my saved passkey.

### Actual Result

The popup displays an error: `Could not complete biometrics.`

### Screenshots or Videos

<img width="360" alt="screenshot" src="https://github.com/bitwarden/clients/assets/67524957/8c4ffd4f-886c-4013-99f7-408eb06f1d70">

### Additional Context

I can confirm that the `Use master password` option works as expected.

This error seemingly only occurs if the browser extension is already unlocked when I attempt to log in. If the extension is locked, I am prompted to unlock the vault using biometrics (which succeeds) before the `Log in with passkey?` popup is shown. Once I click `Confirm` I am logged in as expected.

### Operating System

macOS

### Operating System Version

Sonoma 14.3.1

### Web Browser

Brave

### Browser Version

1.67.119

### Build Version

2024.6.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by gokussjx** [comment]
Same issue on both Firefox and Brave on macOS. Surprisingly, Safari is fine, likely due to the extension being an older one?

**Comment by SergeantConfused** [comment]
Hello @ethansaxenian and everyone else,

Thank you all for your input. I was able to reproduce this behaviour and have flagged it to the Engineering department.
Please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you all again,

**Comment by just5ky** [comment]
### Windows
Browser: Brave `Version 1.67.119 Chromium: 126.0.6478.114 (Official Build) (64-bit)`
Extension version:  `2024.6.2`
![image](https://github.com/bitwarden/clients/assets/71321862/c85918c8-d8c2-434c-ae4b-9d8ad00e0674)

Desktop App version: 
![image](https://github.com/bitwarden/clients/assets/71321862/c845ff9a-d1d2-4cd8-8ad7-f0423abbf29e)

### MacOS
Browser: Brave `Version 1.67.119 Chromium: 126.0.6478.114 (Official Build) (arm64)`
Extension version:  `2024.6.2`
Desktop App version: 

---
#### User Verification: Discouraged
![Recording 2024-06-22 at 21 55 40](https://github.com/dani-garcia/vaultwarden/assets/71321862/e9321a8a-ebd8-484e-af9d-9de2268af4c6)

#### User Verification: Preferred
![Recording 2024-06-22 at 21 57 19](https://github.com/dani-garcia/vaultwarden/assets/71321862/31fd231f-918e-46e5-9962-3927c95bbc1c)

**Comment by gokussjx** [comment]
@SergeantConfused I'll amend my statement. I *think* I saw it occur a few times in Firefox, but can't reproduce it any more, hence, Brave is the only one having issues.

**Comment by just5ky** [comment]
> @SergeantConfused I'll amend my statement. I _think_ I saw it occur a few times in Firefox, but can't reproduce it any more, hence, Brave is the only one having issues.


It seems to be to only happening on Chrome Web Store version of the extension

TLDR: works fine in Firefox, Edge with edge web store extension instead of Chrome web store.


Here's the test that I conducted:

#### OS
- Windows 11 IOT LTSC

#### Desktop App
![image](https://github.com/bitwarden/clients/assets/71321862/7a575183-8150-480b-83c8-ffdfaab52b48)

Tested on webauthn.io with `User Verification : Preferred`

#### Browsers
- Edge with Edge web store extension
![image](https://github.com/bitwarden/clients/assets/71321862/6c84eb0a-a49a-4905-b41f-f96ced1f7fa1)


- Edge with Chrome Web Store extension
![image](https://github.com/bitwarden/clients/assets/71321862/2379dc72-b4bb-46f7-83a8-ea89707c72de)

- Chrome
![image](https://github.com/bitwarden/clients/assets/71321862/b6bc9263-63ec-4b6d-8ed8-3adfa478978e)

- Brave
![image](https://github.com/bitwarden/clients/assets/71321862/24e79ab9-babe-4342-87d9-8a9c6eea6cb5)

- Firefox
![image](https://github.com/bitwarden/clients/assets/71321862/33a1091c-eb27-4073-bbd6-d0d319d49404)

**Comment by motamedn** [comment]
Same occuring in Chrome. Can confirm that "locking" chrome extension, then logging in with passkey works. If vault is unlocked, biometrics log in does not work. Can still use master password option.

**Comment by martinjgrunwald** [comment]
Same problem here.
Chrome Extension 2024.6.2

**Comment by tofr** [comment]
This is.... concerning. I too am having this issue on Vivaldi and Firefox on linux mint 21.1. Passkeys have been wonky for months with BW. Locked out of several accounts at the moment...

**Comment by reubenyap** [comment]
Yes can confirm this issue on Brave with extension Version: 2024.6.1

**Comment by bin101** [comment]
@gbubemismith Are you aware of this?

**Comment by gbubemismith** [comment]
@bin101, Thank you for bringing this to my attention. I would look into this

**Comment by trmartin4** [comment]
Hello,

We had already documented this defect and verified that [this](https://github.com/bitwarden/clients/pull/10050) pull request will address this issue.

This will be included in the next release candidate for our extensions.  I cannot commit to a date for this yet, but I will close this ticket once released.

**Comment by trmartin4** [comment]
This issue was resolved with the `2024.7.1` release of the Bitwarden extension, which has been submitted to stores for approval and will be released as soon as approved by each store.

We are closing this issue, but please feel free to re-open or create a new issue if you continue to experience problems.  We want to make sure that these frustrating issues can be identified and resolved as quickly as possible.
